wrappers/Python/setup.py

```diff
@@ -169,6 +181,26 @@ def recursive_collect_includes():
                 cmake_config_args += ['-G', '"Visual Studio 15 2017 Win64"']
             else:
                 raise ValueError('cmake_bitness must be either 32 or 64; got ' + cmake_bitness)
+        elif cmake_compiler == 'vc16':
+            cmake_build_args = ['--config', '"Release"']
+            if cmake_bitness == '32':
+                cmake_config_args += ['-G', '"Visual Studio 16 2019"', '-A',
+                                      'x86']
+            elif cmake_bitness == '64':
+                cmake_config_args += ['-G', '"Visual Studio 16 2019"', '-A',
+                                      'x64']
+            else:
+                raise ValueError('cmake_bitness must be either 32 or 64; got ' + cmake_bitness)
+        elif cmake_compiler == 'vc17':
+            cmake_build_args = ['--config', '"Release"']
+            if cmake_bitness == '32':
+                cmake_config_args += ['-G', '"Visual Studio 17 2022"', '-A',
+                                      'x86']
+            elif cmake_bitness == '64':
+                cmake_config_args += ['-G', '"Visual Studio 17 2022"', '-A',
+                                      'x64']
+            else:
+                raise ValueError('cmake_bitness must be either 32 or 64; got ' + cmake_bitness)
```

Comment:
add MSVC 2019 and 2022

Reply:
That's why we have "any" option, is this really needed? I recall having issues with this myself